packages/wonder-stuff-server/src/get-logging-transport.js

```diff
@@ -0,0 +1,94 @@
+// @flow
```

Comment:
Copied from render-gateway but tweaked to remove Google-specific stuff and use `Runtime` enum.